client/client.go

```diff
@@ -0,0 +1,36 @@
+package client
+
+import (
+	"context"
+
+	sdkparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	"github.com/umee-network/umee/v4/sdkclient"
+)
+
+type Client struct {
+	sdkclient.Client
+}
+
+// Initializes a cosmos sdk client context and transaction factory for
+// signing and broadcasting transactions
+func NewClient(
+	chainID,
+	tmrpcEndpoint,
+	grpcEndpoint,
+	accountName,
+	accountMnemonic string,
+	gasAdjustment float64,
+	ecfg sdkparams.EncodingConfig,
```

Comment:
Naming this variable `encodingConfig` would be a lot more readable.

Reply:
Because for someone reviewing or modifying the code for the first time seeing "ecfg" 100 lines down from the declaration doesn't mean anything to me. I would have to scroll up to check what that variable is supposed to be. If there are a couple more variables like that it becomes more unreadable. I wouldn't call it "repeating". I'd call it "not abbreviating past the point of readability". IMO the extra few characters are worth readability and maintainability but this is not a blocker.

Reply:
Using `encCfg` which is also used in Cosmos SDK